test/README.md

```diff
@@ -655,6 +655,7 @@ simpler and faster to work with a minimal environment.
 - `submariner.yaml` - for testing `submariner` deployment
 - `velero.yaml` - for testing `velero` deployment
 - `volsync.yaml` - for testing `volsync` deployment
+- `olm.yaml` - for testing `olm` deployment
```

Comment:
This list is easier to manage if it is sorted by env name. The current list is almost sorted, so lets add the new item in the right place, and sort the entire list.